app/presenters/auction_presenter.rb

```diff
@@ -89,20 +89,24 @@ def bid_count
     bids.size
   end
 
-  def starts_at
+  def formatted_start_time
     DcTimePresenter.convert_and_format(model.start_datetime)
   end
 
-  def ends_at
+  def formatted_end_time
     DcTimePresenter.convert_and_format(model.end_datetime)
   end
 
-  def starts_in
+  def formatted_delivery_deadline
+    DcTimePresenter.convert_and_format(model.delivery_deadline)
+  end
+
+  def relative_start_time
     time_in_human(model.start_datetime)
   end
 
-  def ends_in
-    time_in_human(model.end_datetime)
+  def relative_time_left
+    "#{distance_of_time_in_words(Time.now, model.end_datetime)} left"
```

Comment:
thoughts on `Time.current` (alias for `Time.zone.now`) instead of `Time.now`? The former takes into account the timezone. http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/